### PR TITLE
Bump to v2.4.0

### PR DESCRIPTION
**Pull request by yhcao6**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 ## Changelog
 
+### v2.4.0 (5/9/2020)
+
+**Highlights**
+- Refactor common issues and add trouble shooting (#3530)
+- Support new methods [Yolact](https://arxiv.org/abs/1904.02689), [PAA Assign](https://arxiv.org/abs/2007.08103)
```
Comment (ZwwWayne) [inline docs/changelog.md:7]: YOLOv3

File: docs/changelog.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
 ## Changelog
 
+### v2.4.0 (5/9/2020)
+
+**Highlights**
+- Refactor common issues and add trouble shooting (#3530)
+- Support new methods [SABL](https://arxiv.org/abs/1912.04260), [YOLOv3](https://arxiv.org/abs/1804.02767), and [PAA Assign](https://arxiv.org/abs/2007.08103)
+- Support Batch Inference
+- Start to publish `mmdet` package to PyPI since v2.4.0
+- Switch model zoo to aliyun
```
Comment (hellock) [inline docs/changelog.md:10]: Switch model zoo to download.openmmlab.com

File: docs/changelog.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
 ## Changelog
 
+### v2.4.0 (5/9/2020)
+
+**Highlights**
+- Refactor common issues and add trouble shooting (#3530)
```
Comment (hellock) [inline docs/changelog.md:6]: Fix lots of issues/bugs and reorganize the trouble shooting page.

File: docs/install.md [hunk 4]
```diff
@@ -73,6 +73,7 @@ pip install mmcv-full
 | MMDetection version |    MMCV version     |
 |:-------------------:|:-------------------:|
 | master              | mmcv-full>=1.0.5, <=1.1.1|
+| 2.4.0               | mmcv-full>=1.0.5, <=1.1.1|
```
Comment (hellock) [inline docs/install.md:76]: outdated

**Review by hellock — APPROVED** [review]

**Pull request merged**